.gitignore

```diff
@@ -23,4 +23,5 @@ logs
 **/too_big.txt
 chain-tests-report/
 .nyc_output
-coverage
\ No newline at end of file
+coverage
+!src/server/controllers/repository
```

Comment:
Why is this?

Reply:
because of the existance of repository

Reply:
Why is there a repository under controllers? and what does it mean the repository exists under the path controller? The contract repository (i.e. the files of the verified contracts) should not be inside the sourcify source code folder.

Reply:
I don't know if it's still the case but it used to be a repository folder in the the project. in the .gitignore file is ignored every folder called repository. I didn't check if it is still the case. But in order to allow the controller `repository` folder in the repository I had to exclude it from the gitignore.

Reply:
Ah right my bad, we actually do have a `controllers/repository` folder. Makes sense now